scripts/sync_character_langs.py

```
@@ -0,0 +1,212 @@
+        key = zh_to_key.get(zh)
+        if key is None:
+            key = ZH_KEY_MAP.get(zh)
+        if key is None:
+            manual.append(zh)
+            continue
+        is_new = key not in lang
+        node = lang.setdefault(key, {})
+        if is_new:
+            node.setdefault("zh_CN", {}).setdefault("string", wiki_vals.get("zh_CN", zh))
+            node.setdefault("zh_TW", {}).setdefault("string", wiki_vals.get("zh_TW", zh))
+            # es/spname 缺失时用官方英文名回退（保证节点完整，wiki 补录后自动覆盖）
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)